addons/website_sale/views/templates.xml

```diff
@@ -259,13 +259,15 @@
                             </div>
                             <t t-else="">
                                 <div class="text-center text-muted oe_product">
```

Comment:
`oe_product` can safely be removed. It is causing an issue where the block is acting like a product block and shows customize options like `Push to top` but it crashes on click as this is not a real product block.

Introduced with https://github.com/odoo/odoo/commit/23f0fb616802cbfc156ad6847e482e7e71f6a070 in 10.0 but this behavior was automatically/indirectly fixed when we switch to BS4 in 12.0 so this line is not needed anymore.

Reply:
Done